Guess game/Program.cs

```diff
@@ -51,39 +56,60 @@ static int StartGame()
                 foreach (KeyValuePair<string, string> riddle in Riddles)
                 {
                     int guessCount = 0;
+                    Console.WriteLine("------------------------------------------------------------------------------");
                     Console.WriteLine(riddle.Key);
 
                     while (guessCount < guessLimit)
                     {
-                        Console.Write("Enter your guess ");
+                       Console.WriteLine();
+                       Console.Write("Įveskite savo spėjimą: ");
                        string guess = Console.ReadLine();
 
                         if (guess == riddle.Value)
                         {
-                            GiveAnswer(correct: true);
+                            GiveAnswer(correct: true, lastAttempt: false);
                             progress += guessLimit - guessCount;
                             break;
                         }
 
                         if (guessCount == guessLimit - 1 & guess != riddle.Value)
                         {
-                            GiveAnswer(correct: false);
+                            GiveAnswer(correct: false, lastAttempt: true);
                             break;
                         }
 
+                        GiveAnswer(correct: false, lastAttempt: false);
                         guessCount++;
                     }
                 }
 
             return progress;
         }
 
-        private static void GiveAnswer(bool correct)
+        private static void GiveAnswer(bool correct, bool lastAttempt)
         {
-            Console.WriteLine(correct ? "Nice you get it " : "You did not get it ");
+            if (correct == true && lastAttempt == false)
+            {
+                Console.ForegroundColor = ConsoleColor.DarkGreen;
```

Comment:
Notice that 3 lines  of code is duplicated 3 times in same method
![image](https://user-images.githubusercontent.com/4118417/90336365-166cfb80-dfe4-11ea-9f4b-d0d1ec081b5a.png) i suggest to remove this duplication, you have 2 options:
1) introduce new method 
2) introduce variables in same method and write text only once